ui/src/app/cohort-review/table-page/table-page.tsx

```diff
@@ -411,6 +412,7 @@ export const ParticipantsTable = withCurrentWorkspace()(
     }
 
     goBack() {
+      triggerEvent('Review Participant List', 'Click', 'Back to cohort - Review Participant List');
```

Comment:
Optional: the "Review Participant List' string is used in a few places, you could factor that out into a const PAGE_NAME

Reply:
Factored out this string as well as several in `detail-header` and `detail-tabs` components.